src/main/java/org/cactoos/collection/CollectionEnvelope.java

```diff
@@ -37,9 +37,15 @@
  * @version $Id$
  * @param <X> Element type
  * @since 0.23
+ * @checkstyle AbstractClassNameCheck (500 lines)
  */
-@SuppressWarnings("PMD.TooManyMethods")
-public class CollectionEnvelope<X> implements Collection<X> {
+@SuppressWarnings(
```

Comment:
Don't suppress this warning. There are no qulice rules against abstract classes.